konf/site.yaml

```diff
@@ -0,0 +1,44 @@
+domain: assets.ubuntu.com
+
+image: prod-comms.docker-registry.canonical.com/assets.ubuntu.com
+
+env:
+  - name: FLASK_APP
+    value: webapp.app
+
+  # Postgres
+  - name: DATABASE_URL
+    secretKeyRef:
+      key: assets-ubuntu
+      name: database-url
+
+  # Swift
+  - name: OS_AUTH_URL
+    secretKeyRef:
+      key: assets-ubuntu
+      name: os-auth-url
+
+  - name: OS_USERNAME
+    secretKeyRef:
+      key: assets-ubuntu
+      name: os-username
+
+  - name: OS_PASSWORD
+    secretKeyRef:
+      key: assets-ubuntu
+      name: os-password
+
+  - name: OS_AUTH_VERSION
+    value: 1.0
+
+# Overrides for production
+production:
+  replicas: 2
+  nginxConfigurationSnippet: |
```

Comment:
You need this first if in here too. Same for staging.

Reply:
I added a redirection rule for manager

Reply:
I realised now I pointed you to the wrong line, it should be `$host = 'assets.ubuntu.com'` and `$host = 'assets.staging.ubuntu.com'`. We will actually also need the redirect you wrote for `manage.assets.ubuntu.com` so you can leave it in place if you want, but for it to work we also need to add the domain to extrahosts like in [here](https://github.com/canonical-web-and-design/ubuntu.com/blob/e6166785f958af3bffc1e4d56a5a2fa07438c704/konf/site.yaml#L92). I would not do that right now since it might break the existing asset manager app.